Task1.Logger/Logger.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
```

Comment:
Logger should not be duplicated

Reply:
I will fix so that github would show exact project structure as in my VS